packages/styled-docs/components/Header.jsx

```diff
@@ -21,6 +21,7 @@ const Header = React.forwardRef((props, ref) => {
       top={0}
       height="4rem"
       width="100%"
+      zIndex="2"
```

Comment:
`fixed` is defined in `theme.zIndices`. Do you think it reasonable to use `fixed` instead of numeric value?

```js
const zIndices = {
  hide: -1,
  auto: 'auto',
  base: 0,
  dropdown: 1000,
  sticky: 1100,
  fixed: 1200,
  overlay: 1300,
  drawer: 1400,
  modal: 1500,
  popover: 1600,
  toast: 1700,
  tooltip: 1800,
};
```

Reply:
I forgot that zIndex has variable, I think we can just use the "fixed".